### PR TITLE
Grid component new

### PR DESCRIPTION
**Pull request by flozia**
Ready for testing in the storybook only!

### CODE REVIEW

File: src/Grid/index.js [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    const hasBreakpoint = props.xs || props.sm || props.md || props.lg;
+    const classes = classNames('fb-col', props.className, colBaseClasses, {
+        'fb-flex': !props.noFlex && props.flex !== undefined,
+        'reverse': props.reverse,
+        [classMap.xs]: !hasBreakpoint
+    });
+
+    const Tag = props.tagName || 'div';
+    return (
+        <Tag
```
Comment (TN1ck) [inline src/Grid/index.js]: Better to use `React.createElement`

File: src/Grid/index.js [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+    const colBaseClasses = Object.keys(props).filter((key) => { // filter props that match any item in classMap
+        return classMap[key];
+    }).map((key) => { // take care of valid classnames
+        const colBase = !(typeof props[key] === 'boolean') ? (classMap[key] + '-' + props[key]) : classMap[key];
+        return colBase;
+    }).join(' ').toString();
+
+    const hasBreakpoint = props.xs || props.sm || props.md || props.lg;
+    const classes = classNames('fb-col', props.className, colBaseClasses, {
+        'fb-flex': !props.noFlex && props.flex !== undefined,
```
Comment (TN1ck) [inline src/Grid/index.js]: Flex is not specified in the propTypes

File: src/Grid/index.js [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+        const value = props[toCamelCase(modificatorKey)];
+        return `fb-${modificatorKey}-${value}`;
+    });
+
+    const classes = classNames(props.className, modificators, {
+        // Col
+        [`fb-col-${props.column}`]: props.column,
+        // Row
+        'fb-row': !props.noFlex,
+        [`fb-row-${props.row}`]: props.row,
```
Comment (TN1ck) [inline src/Grid/index.js]: `.row` is not specified in the propTypes

File: src/Grid/index.js [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+        'fb-row--no-gutter': props.noGutter,
```
Comment (TN1ck) [inline src/Grid/index.js]: `.noGutter` is not specified in the propTypes

File: src/Grid/index.js [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+        'fb-row--gutter-top-bottom': props.gutterTopBottom
```
Comment (TN1ck) [inline src/Grid/index.js]: `.gutterTopBottom` is not specified in the propTypes

File: src/Grid/index.js [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+    });
+
+    const Tag = props.tagName || 'div';
+    return (
+        <Tag
```
Comment (TN1ck) [inline src/Grid/index.js]: Here as well!

File: src/Grid/index.js [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+ * @param {Object} props      - The props for the component
+ * @returns {React.Component} - Component to be returned
+ */
+function Grid (props) {
+    const containerClass = props.fluid ? 'fb-container-fluid' : 'fb-container';
+    const classes = classNames(props.className, containerClass);
+    const Tag = props.tagName || 'div';
+
+    return (
+        <Tag className={classes}>
```
Comment (TN1ck) [inline src/Grid/index.js]: Use `React.createElement`

**Comment by onlyahost** [comment]
@flozia 
I will correct the issues Tom mentioned and merge it.
It would be nice if we would start integrating it soon to check if it works out. :-)

**Comment by onlyahost** [comment]
@TN1ck Thanks for the Review - fixed this stuff.

**Pull request merged**